js/game.js

```diff
@@ -0,0 +1,10 @@
+class Game {/**Evil comments with my freaky peasant language skills*/
+    constructor() { 
+      this.gameRound = this.gameRound /**This is like a status, how many times has been played, gamerond act as iterator */
```

Comment:
Šis neko nedod, respektīvi this.gameRound būs undefined.
Komentārs lieks.

Reply:
ar gameRound biju domājis, kura spēles partija jau ir izspēlēta pēc kārtas no esošās kāršu kavas